content/ja/community/code-of-conduct.md

```diff
@@ -0,0 +1,19 @@
+---
+title: コミュニティ
+layout: basic
+cid: community
+css: /css/community.css
+---
+
+<div class="community_main">
+<h1>Kubernetesコミュニティ行動規範</h1>
+
+Kubernetesは、<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/jp.md">CNCF行動規範</a>に従います。
+以下のCNCFの行動規範のテキストは、<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit da91d9e</a>から複製したものです。この掲示が古くなっていることに気がついた場合には、<a href="https://github.com/kubernetes/website/issues/new">issueを作成</a>してください。
```

Comment:
nit: https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct-languages/jp.md

Reply:
Also `da91d9e` != `214585e` (is that wrong upstream too?)

Reply:
Yes, `214585e` is the latest one. I took `da91d9e` from the commit history of Japanese translation but it was a little old.